.github/workflows/linters-readme.yml

```diff
@@ -0,0 +1,54 @@
+# We need a separate workflow file so failures on the README are not reported on
+# all PRs, only on the ones that update the README.
+name: Linters (README.md)
+
+on:
+  # We need `pull_request_target` to access the secrets
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths: [README.md]
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: '3.12'
+  NODE_VERSION: lts/*
+
+permissions:
+  contents: read
+
+jobs:
+  lint-readme-lists:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  # v4.1.7
+        with:
+          persist-credentials: false
+          ref: ${{ github.event.repository.default_branch }}
+      - name: Get team members
+        id: team_members
+        run: |
+          get_list_members() {
+            TEAM="$1"
+            QUOTE='"'
+            gh api "/orgs/$GITHUB_REPOSITORY_OWNER/teams/$TEAM/members" -X GET -f per_page=100 --jq "map(.login) | ${QUOTE}${TEAM}=\(tojson)${QUOTE}"
+          }
+
+          get_list_members "collaborators" >> "$GITHUB_OUTPUT"
+          get_list_members "issue-triage" >> "$GITHUB_OUTPUT"
+          get_list_members "tsc" >> "$GITHUB_OUTPUT"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+      - name: Get modified README
+        if: github.event_name != 'push'
+        run: curl "https://patch-diff.githubusercontent.com/raw/$GITHUB_REPOSITORY/pull/${PULL_ID}.diff" | git apply --include=README.md
+        env:
+          PULL_ID: ${{ github.event.pull_request.number }}
+      - run: tools/lint-readme-lists.mjs "$TEAMS"
```

Comment:
```suggestion
      - run: mv node/README.md README.md && tools/lint-readme-lists.mjs "$TEAMS"
```

Reply:
The sparse-checkout will checkout in the `node` directory, so the file would have to be moved (if the suggest is accepted)